Modix/Modix.csproj

```diff
@@ -12,21 +12,20 @@
     <NpmBuild Condition=" '$(Configuration)' == 'Debug' ">build-dev</NpmBuild>
     <NpmBuild Condition=" '$(NpmBuild)' == '' ">build</NpmBuild>
     <DevServerRunning>false</DevServerRunning>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <Folder Include="wwwroot\" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="AspNet.Security.OAuth.Discord" Version="2.0.1" />
-    <PackageReference Include="Discord.Net.WebSocket" Version="2.1.1" />
+    <PackageReference Include="AspNet.Security.OAuth.Discord" Version="3.0.0" />
```

Comment:
Versions would be easier to handle if they were centrally managed.

You can define these as `<PackageReference Include"Name" />` and then, in a central `Directory.Build.targets`  do `<PackageReference Update="Name" Version="..." />`

This ensures you have one place to update for version upgrades and makes it easy to manage everything at once or to document places that differ.

Reply:
That's really cool! I didn't know you could do that

Reply:
You can see an example of how I do this in TerraFX here: <https://github.com/terrafx/terrafx/blob/master/Directory.Build.targets>

Some other repos (like dotnet/project-system) do this as well.

Reply:
Didn't know this existed, has been integrated as part of the latest commit, 👍